### PR TITLE
Add Docker article

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [x] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/docs-conceptual/install/PowerShell-in-Docker.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Using  PowerShell in Docker
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/install/PowerShell-in-Docker.md]:
```suggestion
title: Using PowerShell in Docker
```

File: reference/docs-conceptual/install/PowerShell-in-Docker.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+description: How to use PowerShell that is preinstalled in a Docker image.
+ms.devlang: powershell
+ms.topic: conceptual
+ms.date: 03/03/2020
+---
+
+# Using PowerShell in Docker
+
+We are publishing Docker images with PowerShell preinstalled. This article shows you how to get
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/install/PowerShell-in-Docker.md]:
```suggestion
We publish Docker images with PowerShell preinstalled. This article shows you how to get
```

File: reference/docs-conceptual/install/PowerShell-in-Docker.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+By requesting and using the Container OS Image for Windows containers, you acknowledge,
+understand, and consent to the Supplemental License Terms available on Docker hub:
+
+- [Window Server Core](https://hub.docker.com/r/microsoft/windowsservercore/)
+- [Nano Server](https://hub.docker.com/r/microsoft/nanoserver/)
+
+### Telemetry
+
+By default, PowerShell collects the OS description and the version of PowerShell (equivalent to
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/install/PowerShell-in-Docker.md]: This is no longer accurate with PS7.  Should just say something like: By default, PowerShell collects limited telemetry without personally identifiable information to help aid development of future versions of PowerShell.

File: reference/docs-conceptual/install/PowerShell-in-Docker.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+## Legal and Licensing
+
+PowerShell is licensed under the [MIT license][].
+
+### Windows Docker File and Image Licenses
+
+By requesting and using the Container OS Image for Windows containers, you acknowledge,
+understand, and consent to the Supplemental License Terms available on Docker hub:
+
+- [Window Server Core](https://hub.docker.com/r/microsoft/windowsservercore/)
```
Comment (chasewilson) [inline reference/docs-conceptual/install/PowerShell-in-Docker.md]: Why not maintain consistency and use the other localized link system as all the other links on the page?

**Pull request merged**